.github/workflows/update_bpmn_visualization_version.yml

```diff
@@ -30,17 +30,17 @@ jobs:
       - uses: actions/checkout@v2
       - name: Update examples
         run: ./scripts/update-lib-version.bash ${{ env.VERSION }}
-      - name: Delete old Demo
-        run: rm -rf demo
-      - name: Download Demo ${{ env.VERSION }}
+      - name: Delete old Load and Navigation demo
+        run: rm -rf demo/load_and_navigation
```

Comment:
⚠️ we use kebab-case for directory name everywhere in the repository, so for consistency `load_and_navigation` --> `load-and-navigation`